### PR TITLE
Feature/upgrade to Bootstrap 5

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Upgrade of bootstraps to latest version.
- Replacement of dropdown select component.
- Re-organization page settings.
- Improvements to style code.
- Improvements to review code panel.
- Some improvements to theming.

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Client/css/java.scss [hunk 1]
```diff
@@ -1,47 +0,0 @@
-.icon-assembly {
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Client/css/java.scss:1]: Why is this removed? This was added to support Language specific icons. are we supporting it in different way?

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Client/css/java.scss:1]: Never mind. I see how you are handling it.

**Review by praveenkuttappan — APPROVED** [review]
Looks good

File: src/dotnet/APIView/APIViewWeb/Models/ReviewRevisionModel.cs [hunk 2]
```diff
@@ -44,6 +44,17 @@ public string Author
             set => _author = value;
         }
 
+        [JsonIgnore]
+        public string DisplayNameShort
```
Comment (chidozieononiwu) [inline src/dotnet/APIView/APIViewWeb/Models/ReviewRevisionModel.cs:48]: @praveenkuttappan
